corehq/apps/registration/forms.py

```diff
@@ -46,9 +46,6 @@ class NewWebUserRegistrationForm(DomainRegistrationForm):
     """
     Form for a brand new user, before they've created a domain or done anything on CommCare HQ.
     """
-    full_name = forms.CharField(label=_('Full Name'),
```

Comment:
Good to go on required now, for some reason I thought it was optional on
prod currently, but just tested and it's not.

Could be fun to A/B test optional vs. required, but comms through hubspot
can't handle that.

Thanks for pinging

On Tue, Sep 22, 2015 at 2:23 PM, Jenny Schweers <notifications@github.com>
wrote:

> In corehq/apps/registration/forms.py
> <https://github.com/dimagi/commcare-hq/pull/8202#discussion_r40123745>:
>
> > @@ -46,9 +46,6 @@ class NewWebUserRegistrationForm(DomainRegistrationForm):
> >      """
> >      Form for a brand new user, before they've created a domain or done anything on CommCare HQ.
> >      """
> > -    full_name = forms.CharField(label=_('Full Name'),
>
> @dimagi/product <https://github.com/orgs/dimagi/teams/product> ping,
> should name should be optional or required?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/8202/files#r40123745>.
>